### DISCUSSION TITLE
training loss

### DISCUSSION THREAD

**Discussion by JohnnMa** [PixArt-alpha/PixArt-alpha#165] [General]
When I use the SAM config to train  SAM dataset, the loss quickly stabilizes at around 0.14 and no longer decreases. Is this normal?
